commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -45,8 +45,12 @@
         <section class="hosted-tone--dark">
             <div class="host hosted__container--full">
                 <div class="u-responsive-ratio u-responsive-ratio--hd">
-                    @if(page.video.youTubeHtml.isDefined){
-                        @Html(page.video.youTubeHtml.get)
+                    @if(page.video.youtubeId.isDefined){
+                        <iframe id="hosted-video-@{page.video.youtubeId}"
+                        class="js-hosted-youtube-video"
+                        src="https://www.youtube.com/embed/@{page.video.youtubeId}?modestbranding=1&showinfo=0&rel=0&enablejsapi=1"
```

Comment:
you need to add `&origin=theguardian.com` see [origin](https://developers.google.com/youtube/player_parameters#origin)